cliquet/resource.py

```diff
@@ -282,7 +290,8 @@ def collection_delete(self):
         filters = self._extract_filters()
         deleted = self.collection.delete_records(filters=filters)
 
-        return self.postprocess(deleted)
+        action = len(deleted) > 0 and ACTION_DELETE or ACTION_READ
```

Comment:
Reading further, I feel a bit embarrassed that you consider ACTION_READ as being: not sending event.